### PR TITLE
docs: improve Tauri integration

### PR DESCRIPTION
**Pull request by nevthereal**
# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

I saw that the tauri integrations docs weren't explaining how to set up a tauri app with qwik. I found it to be an extra hurdle to go over to the tauri docs, so I added the docs from the official website to make the process a bit faster.

# Use cases and why

You want to swiftly set up Tauri. Having the full docs in one place is a bit more efficient than jumping from page to page.

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/QwikDev/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/docs/integrations/tauri/index.mdx [hunk 1]
```diff
@@ -14,3 +14,52 @@ created_at: '2023-05-14T08:01:52Z'
...
+To install Tauri via npm run:
+
+```shell
+npm install -D @tauri-apps/cli
+```
+
+In your `package.json`, add this script:
+
+```json title="package.json"
+```
```
Comment (gioboa) [inline packages/docs/src/routes/docs/integrations/tauri/index.mdx:38]: Thanks for your help @nevthereal 
did you miss the script here?

**Review by gioboa — CHANGES_REQUESTED** [review]

File: packages/docs/src/routes/docs/integrations/tauri/index.mdx (hunk 1 above)
Comment (nevthereal) [reply packages/docs/src/routes/docs/integrations/tauri/index.mdx:38]: oh yes. should I add them real quick?

Comment (nevthereal) [reply packages/docs/src/routes/docs/integrations/tauri/index.mdx]: added!

**Review by gioboa — APPROVED** [review]
Thanks :clap: @nevthereal

**Pull request merged**